From 8d0f03242b3519c07c8e921be01ddc52b0064b5f Mon Sep 17 00:00:00 2001
From: "Brett T. Warden" <brett.t.warden@intel.com>
Date: Wed, 23 Oct 2024 11:19:55 -0700
Subject: [PATCH] Compress firmware with ZST by default

---
 Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
--- linux-firmware-20241210/Makefile~	2024-12-10 16:54:41.000000000 +0000
+++ linux-firmware-20241210/Makefile	2025-01-06 15:27:32.651259108 +0000
@@ -29,7 +29,8 @@
 dedup:
 	./dedup-firmware.sh $(DESTDIR)$(FIRMWAREDIR)
 
-install:
+install: install-zst
+install-uncompressed:
 	@if [ -n "${COPYOPTS}" ]; then \
 		echo "COPYOPTS is not used since linux-firmware-20241017!"; \
 		echo "You may want to use install{-xz,-zst} and dedup targets instead"; \
